CanvasApi.Client/Admins/AdminApiClient.cs

```diff
@@ -0,0 +1,37 @@
+﻿using CanvasApi.Client._Base;
+using CanvasApi.Client.Admins.Models;
+using CanvasApi.Client.Extentions;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanvasApi.Client.Admins
+{
+    public class AdminApiClient : ApiClientBase, IAdminApiClient
+    {
+        public AdminApiClient(CanvasApiClient parent) : base(parent) { }
+
+        public async Task<IAdmin> Make(long accountId, Action<IAdminMake> admin) =>
+            await ApiClient.ApiOperation<Admin, IAdminMake>(
```

Comment:
Maybe we should use an industry term such as Promote since the API is promoting a current user to Admin status.

Reply:
I will make the change. I was trying to keep consistent with the API documentation, but I agree the terminology does not sound right